app/views/govuk_publishing_components/components/_step_by_step_nav_header.html.erb

```diff
@@ -21,15 +21,16 @@
 
   local_assigns[:margin_bottom] ||= 0
   shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
-  classes = "gem-c-step-nav-header #{shared_helper.get_margin_bottom}"
+  classes = %w(gem-c-step-nav-header)
+  classes << shared_helper.get_margin_bottom if local_assigns[:margin_bottom] != 0
```

Comment:
I think we could actually remove line 22, and change this line to be `classes << shared_helper.get_margin_bottom if local_assigns[:margin_bottom]`

Reply:
That looks good thanks, i've now updated as above.